torch_xla/csrc/helpers.cpp

```diff
@@ -11,6 +11,8 @@
 
 namespace torch_xla {
 
+bool XlaHelpers::s_use_full_mat_mul_precision = false;
```

Comment:
Maybe a set/get access was better, but no need to do it now.

Reply:
Done.